packages/paste-website/src/component-examples/StatusPatternExamples.tsx

```diff
@@ -0,0 +1,511 @@
+    <TBody>
+      <Tr>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="success"
+            />
+            <Box>
+              <Text as="div">Successfully exported</Text>
+              <Text as="div" color="colorTextWeak">Tuesday, January 2, 2021</Text>
+            </Box>
+          </Box>
+        </Td>
+        <Td>
+          <strong>Contacts who did not click</strong>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="success"
+            />
+            <Box>
+              <Text as="div">Successfully exported</Text>
+              <Text as="div" color="colorTextWeak">Tuesday, February 2, 2021</Text>
+            </Box>
+          </Box>
+        </Td>
+        <Td>
+          <strong>All contacts</strong>
+        </Td>
+      </Tr>
+    </TBody>
+  </Table>
+  <Box marginTop="space40">
+    <Heading as="h2" variant="heading20" marginTop="space40">
+      IDs
+    </Heading>
+  </Box>
+  <Table>
+    <THead>
+      <Tr>
+        <Th>ID</Th>
+        <Th>Unique name</Th>
+        <Th>Region</Th>
+        <Th>Status</Th>
+        <Th>Order Date</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>90294808908109537</Td>
+        <Td>WE902990c21gjioGasd</Td>
+        <Td>North America</Td>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="active"
+            />
+            <Text as="div">Active</Text>
+          </Box>
+        </Td>
+        <Td>2020-10-15</Td>
+      </Tr>
+      <Tr>
+        <Td>90294808908872345</Td>
+        <Td>WE928471c21gjioGasd</Td>
+        <Td>North America</Td>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessNeutralIcon
+              color="colorTextIconNeutral"
+              decorative={false}
+              title="active"
+            />
+            <Text as="div">Ready</Text>
+          </Box>
+        </Td>
+        <Td>2020-10-15</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</>
+`.trim();
+
+export const tableWithPlainTextExample = `
+<>
+  <Heading as="h2" variant="heading20">
+    Reboots
+  </Heading>
+
+  <Table>
+    <THead>
+      <Tr>
+        <Th>Title</Th>
+        <Th>Media</Th>
+        <Th>Status</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>Bewitched</Td>
+        <Td>Series</Td>
+        <Td>
+          <Box display="flex">
+            <ProcessWarningIcon color="colorTextIconWarning" decorative={false} title="progress halted" />
+            <Text marginLeft="space20">Progress Halted</Text>
```

Comment:
```suggestion
            <Text marginLeft="space20">Progress halted</Text>
```

Reply:
We use sentence casing for things like this across our products, so just wanted to update this example to reflect that.